app/uk/gov/hmrc/trackingconsentfrontend/assets/test/config/config.spec.ts

```diff
@@ -0,0 +1,72 @@
+import '@testing-library/jest-dom/extend-expect'
+import { getLocalBaseUrl } from '../../src/config/config'
+
+describe('config', () => {
+  beforeEach(() => {
+    document.getElementsByTagName('html')[0].innerHTML = `<html>
+    <head>
+        <script src="/another-script.js"></script>
+        <script data-id="tracking-consent-frontend" data-port="54321" src="/bundle.js"></script>
```

Comment:
In what scenario would it be desirable to point to a port other than the one tracking consent is being loaded from?

Reply:
It is needed for acceptance testing. In Jenkins, the tracking consent service runs under a different port.